tests/multi_gcov_aggregator.py

```diff
@@ -0,0 +1,145 @@
+# Script to merge coverage files generated by gcov
+# Author: Noe Brucy, Tine Colman
+# Known issues:
+# - in rt_spectra, gcov identified some variable declarations as executed.
+# - in turb_force_utils, stop is identified as a branch
+
+import os
+import re
+from collections import defaultdict
+from glob import glob
+
+class GCovParser:
+    def __init__(self):
+        # Dictionary to store data as {source_file: {line_number: (execution_count, line_content)}}
```

Comment:
Isn't it instead?
```suggestion
        # Dictionary to store data as {source_file: {line_number: (execution_count, line_number, line_content)}}
```
If so, why `""` as a default line number rather than some integer (like -1)?

Reply:
No, the 3th entry is a list of the tests in which the line is executed. For now I disabled printing this because the line gets very long for lines that are touched by many tests.